sdcm/cluster.py

```diff
@@ -2377,23 +2377,34 @@ def add_install_prefix(self, abs_path):
         }
         return checklist.get(abs_path, INSTALL_DIR + abs_path)
 
+    def _service_cmd(self, service_name: str, cmd: str, timeout: int = 500, ignore_status=False):
+        opts = {'timeout': timeout, 'ignore_status': ignore_status}
+        if self.is_ubuntu14():
+            cmd = f'sudo service {service_name} {cmd}'
```

Comment:
why do we need sudo here, and not on `systemctl`?
i would suggest to change the line 2386 to `self.remoter.sudo()` as IIRC `systemctl` also requires `sudo` permission

Reply:
i understand it now from @bentsi 's comments

Reply:
it is actually selected here: `sdcm.cluster.BaseNode.wrap_cmd_with_permission`